common-lib/src/main/resources/application.conf

```diff
@@ -109,3 +109,5 @@ metadata.templates = []
 #  label: "chargeable",
 #  chargeable: true
 # }
+
+restrictDownload = true
```

Comment:
should this be the default?

Reply:
If this controls functionality introduced in this here PR – no. The default should be current behaviour.

Reply:
cheers - will update 👍